src/BlazorUI/Bit.BlazorUI.Extras/Scripts/Extras.ts

```
@@ -56,5 +56,41 @@ namespace BitBlazorUI {
+            const notAddedStylesheets = stylesheets.filter(s => !allStylesheets.find(as => as.includes(s)));
+
+            if (notAddedStylesheets.length == 0) return Promise.resolve();
+
+            const promise = new Promise(async (res: any, rej: any) => {
+                try {
+                    await Promise.all(notAddedStylesheets.map(addStylesheet));
+                    res();
+                } catch (e: any) {
+                    rej(e);
+                }
+            });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)